processFolder.py

```diff
@@ -0,0 +1,56 @@
+# Tayo Elelu
+# Created 10/14/16
+# CS 4365 - Software Engr II
+# Image Processing Module
+
+# Using Pillow library
+# Given a directory, this module will analyze the folder.
+# It will go thru all images, resize
+# them to 75x75 convert them to grayscale,
+# and save the images for later use/output an array of them.
+
+################################################################
+
+#Import needed Pillow libraries
+from PIL import Image # image libraries and operations
+import os, sys        # directories 

```

Comment:
This is _literally_ the example for what **not** to do when importing in Python according to the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html#Imports_formatting)